packages/webview_flutter/webview_flutter/example/ios/RunnerUITests/FLTWebViewUITests.m

```diff
@@ -5,6 +5,30 @@
 @import XCTest;
 @import os.log;
 
+static UIColor* getPixelColorInImage(UIImage* image, int x, int y) {
+  CFDataRef pixelData = CGDataProviderCopyData(CGImageGetDataProvider(image.CGImage));
+  const UInt8* data = CFDataGetBytePtr(pixelData);
+
+  int imageWidth = floor(image.size.width * image.scale);
```

Comment:
This is not safe; you want the image's stride, which is `CGImageGetBytesPerRow`.